### ISSUE TITLE
types don't match module format

### ISSUE THREAD

**Issue by jefbarn** [caub/pg-tsquery#25]
Ran into a weird issue with this module after switching typescript to the 'node16' module format.
It boils down to the code is exporting a function as a cjs module, but the index.d.ts definitions are written as if it's esm.

error TS2349: This expression is not callable.
  Type 'typeof import(".../node_modules/pg-tsquery/index")' has no call signatures.

I think the fix would be something like this in index.d.ts instead of the export default:
```typescript
declare function tsquery(options?: TsqueryOptions): (str: string) => string;
export = tsquery
```

For now a work around is this goofy thing:
```typescript
import { default as Tsquery } from 'pg-tsquery'

type TsQueryFunc = (options?: Tsquery.TsqueryOptions) => (str: string) => string
const tsqueryParser = (Tsquery as unknown as TsQueryFunc)()
```

The typescript guys built this little tool that seems to agree with my analysis:
https://arethetypeswrong.github.io/?p=pg-tsquery%408.4.1

**Comment by caub** [comment]
does it work if you use this?
```js
import { Tsquery } from 'pg-tsquery';

const parser = new Tsquery(/* options can be passed to override the defaults */);
```

**Comment by jefbarn** [comment]
TS2614: Module  'pg-tsquery'  has no exported member  Tsquery . Did you mean to use  import Tsquery from "pg-tsquery"  instead?
The published npm module doesn't have Tsquery class defined. 

Tried loading the version from github and got this from node:
import { Tsquery } from 'pg-tsquery';
         ^^^^^^^
SyntaxError: Named export 'Tsquery' not found. The requested module 'pg-tsquery' is a CommonJS module, which may not support all module.exports as named exports.

The issue is if you use the node16 module format to use native es modules in node it won't work because the typescript definition is trying to say this is an es module but it's actually cjs.

**Comment by jefbarn** [comment]
Created a pull request that should match the actual exported code. Take a look.
https://github.com/caub/pg-tsquery/pull/26

**Comment by caub** [comment]
Thanks, I'm not so familiar with TS, so if this fixes the type issues, I'll merge it, thanks again

**Comment by jschuur** [comment]
Will you do a new npm release with this fix? The current 8.4.1 release doesn't have this yet.

**Comment by caub** [comment]
done sorry
